tests/fixtures/__snapshots__/simple1bottom.snap.svg

```diff
@@ -0,0 +1,7 @@
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" stroke-linecap="round" stroke-linejoin="round" stroke-width="0" fill-rule="evenodd" clip-rule="evenodd" id="NDmzwLDbaDfQ_bottom" viewBox="0 -20000 20000 20000" width="20in" height="20in"><defs><style>.NDmzwLDbaDfQ_fr4 {color: #666666;}
```

Comment:
this just looks like a gray square  on github, did something go wrong?

Reply:
It has always been like that